demo/gemini_audio_video/app.py

```diff
@@ -134,56 +134,6 @@ async def shutdown(self) -> None:
     },
 )
 
-css = """
-#video-source {max-width: 600px !important; max-height: 600 !important;}
-"""
-
-with gr.Blocks(css=css) as demo:
-    gr.HTML(
-        """
-    <div style='display: flex; align-items: center; justify-content: center; gap: 20px'>
-        <div style="background-color: var(--block-background-fill); border-radius: 8px">
-            <img src="https://www.gstatic.com/lamda/images/gemini_favicon_f069958c85030456e93de685481c559f160ea06b.png" style="width: 100px; height: 100px;">
-        </div>
-        <div>
-            <h1>Gen AI SDK Voice Chat</h1>
-            <p>Speak with Gemini using real-time audio + video streaming</p>
-            <p>Powered by <a href="https://gradio.app/">Gradio</a> and <a href=https://freddyaboulton.github.io/gradio-webrtc/">WebRTC</a>⚡️</p>
-            <p>Get an API Key <a href="https://support.google.com/googleapi/answer/6158862?hl=en">here</a></p>
-        </div>
-    </div>
-    """
-    )
-    with gr.Row() as row:
-        with gr.Column():
-            webrtc = WebRTC(
-                label="Video Chat",
-                modality="audio-video",
-                mode="send-receive",
-                elem_id="video-source",
-                rtc_configuration=get_cloudflare_turn_credentials_async
-                if get_space()
-                else None,
-                icon="https://www.gstatic.com/lamda/images/gemini_favicon_f069958c85030456e93de685481c559f160ea06b.png",
-                pulse_color="rgb(255, 255, 255)",
-                icon_button_color="rgb(255, 255, 255)",
-            )
-        with gr.Column():
-            image_input = gr.Image(
-                label="Image", type="numpy", sources=["upload", "clipboard"]
-            )
-
-        webrtc.stream(
-            GeminiHandler(),
-            inputs=[webrtc, image_input],
-            outputs=[webrtc],
-            time_limit=180 if get_space() else None,
-            concurrency_limit=2 if get_space() else None,
-        )
-
-stream.ui = demo
```

Comment:
Removed this, since it just added the `image` input to the center of the screen. Using only `stream` correctly adds the image input to the sidebar.

Reply:
Makes sense! Can we remove the `WebRTC` import from the top?